superset-websocket/spec/config.spec.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { buildConfig } from '../src/config';
+
+describe('buildConfig', () => {
```

Comment:
Thanks for this PR! I only have two nits.
- Can you rename it to `config.test.ts`? We're trying to make all tests adhere to the `.test `extension. 
- Can you inline the test? We're trying to [avoid unnecessary nesting](https://kentcdodds.com/blog/avoid-nesting-when-youre-testing) to make the test results easier to read.

Reply:
Looks like this just got merged. I'm happy to make these changes. Will follow up with a PR